SU2_CFD/src/numerics_direct_mean.cpp

```diff
@@ -4115,60 +4115,74 @@ CSourceAxisymmetric_Flow::~CSourceAxisymmetric_Flow(void) { }
 void CSourceAxisymmetric_Flow::ComputeResidual(su2double *val_residual, su2double **Jacobian_i, CConfig *config) {
   
   su2double yinv, Pressure_i, Enthalpy_i, Velocity_i, sq_vel;
-  unsigned short iDim;
+  unsigned short iDim, iVar, jVar;
   
   bool implicit       = (config->GetKind_TimeIntScheme_Turb() == EULER_IMPLICIT);
   bool compressible   = (config->GetKind_Regime() == COMPRESSIBLE);
   bool incompressible = (config->GetKind_Regime() == INCOMPRESSIBLE);
   
-  if (Coord_i[1] > 0.0) yinv = 1.0/Coord_i[1];
-  else yinv = 0.0;
-  
-  if (compressible) {
-    sq_vel = 0.0;
-    for (iDim = 0; iDim < nDim; iDim++) {
-      Velocity_i = U_i[iDim+1] / U_i[0];
-      sq_vel += Velocity_i *Velocity_i;
-    }
-    
-    Pressure_i = (Gamma-1.0)*U_i[0]*(U_i[nDim+1]/U_i[0]-0.5*sq_vel);
-    Enthalpy_i = (U_i[nDim+1] + Pressure_i) / U_i[0];
-    
-    val_residual[0] = yinv*Volume*U_i[2];
-    val_residual[1] = yinv*Volume*U_i[1]*U_i[2]/U_i[0];
-    val_residual[2] = yinv*Volume*(U_i[2]*U_i[2]/U_i[0]);
-    val_residual[3] = yinv*Volume*Enthalpy_i*U_i[2];
-  }
-  if (incompressible) {
-    val_residual[0] = yinv*Volume*U_i[2]*BetaInc2_i;
-    val_residual[1] = yinv*Volume*U_i[1]*U_i[2]/DensityInc_i;
-    val_residual[2] = yinv*Volume*U_i[2]*U_i[2]/DensityInc_i;
+  if (Coord_i[1] > EPS) {
```

Comment:
Here you introduced also tabs. Sorry about that, but we have to be strict on that one to avoid future conflicts.